.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "2.7", "3.6", "3.7", "3.8", "3.9", "3.10" ]
+        python-version: [ "3.2", "3.3", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10" ]
```

Comment:
Unfortunately this doesn't work - the `ubuntu-latest` images now only have Python 3.7 and up. It's possible to use an older image to test on older Python versions (see e.g. [this PR](https://github.com/pypa/flit/pull/615/files)), though I doubt that goes all the way back to 3.2.

We probably have to leave at least some older versions untested. And then decide whether the metadata should claim the minimum version we think is likely to work, or the minimum we actually test. :slightly_smiling_face:

Reply:
I should be more concrete. Looking at the [download stats for PyXDG](https://pypistats.org/packages/pyxdg), it appears that for whatever reason, there are still a lot of downloads on Python 3.6. :shrug:  Let's make that the new minimum supported version, and do the workaround I pointed to in the comment above to test on 3.6. Then whoever is using it can benefit from this fix.

Intriguingly, downloads from Python 2.7 dropped off a cliff in October 2022. I wonder if this was somehow related to the release of pyxdg 0.27, or of Ubuntu 21.10. There's no corresponding rise in downloads from Python 3. :thinking: 

![image](https://user-images.githubusercontent.com/327925/215519065-cf783443-e50b-49bc-ae91-baa0abb78668.png)

Reply:
Sounds good, I'll make those changes this month. You may see this close and reopen in a different PR because (as you can see from the history above) a stale-forks cleaner script mistakenly deleted my fork and closed this, so if I can't resurrect it in place I'll reopen and link here. Sorry in advance for any confusion.

Reply:
Thanks, no problem if it comes as a new PR. I think it should be possible to retrieve this branch to work from even though your repo is deleted, something like this:

```shell
git fetch upstream pull/18/head
git checkout FETCH_HEAD
```